src/sentry/services/hybrid_cloud/user/impl.py

```diff
@@ -88,6 +91,13 @@ def get_by_username(
                 return list(qs.filter(email__iexact=username))
         return []
 
+    def get_organizations(self, *, user_id: int) -> List[RpcOrganizationSummary]:
+        org_ids = OrganizationMemberMapping.objects.filter(user_id=user_id).values_list(
+            "organization_id", flat=True
+        )
+        org_query = OrganizationMapping.objects.filter(organization_id__in=org_ids)
```

Comment:
If we find ourselves doing this kind of query often we might want to add a relationship between OrganizationMemberMapping and OrganizationMapping so that we can do joined loads. Currently there are users with 500+ organization memberships so there could be some benefits from joins.